src/app/retrospective/retrospective-create/retrospective-create.component.html

```diff
@@ -56,8 +57,8 @@ <h2 mat-dialog-title>Create Team Retrospective</h2>
       </mat-card>
     </mat-grid-tile>
   </mat-grid-list>
-  <mat-dialog-actions>
-      <button mat-raised-button mat-dialog-close>Cancel</button>
-      <button mat-raised-button [disabled]="retroFormGroup.invalid || disableButton" (click)="createRetro(retroFormGroup.value)">Create Retro</button>
+  <mat-dialog-actions class="dialog-actions">
+    <button mat-raised-button [disabled]="retroFormGroup.invalid || disableButton" (click)="createRetro(retroFormGroup.value)">Create Retro</button>
```

Comment:
Remove `retro` if the form already justify u are creating retro
Do this everywhere else too